tests/affiliatedcertification/helper/helper.go

```diff
@@ -103,6 +103,21 @@ func DeleteLabelFromInstalledCSV(prefixCsvName string, namespace string, label m
 	return updateCsv(namespace, csv)
 }
 
+func DoesOperatorHaveLabel(prefixCsvName string, namespace string, label map[string]string) bool {
+	csv, err := GetCsvByPrefix(prefixCsvName, namespace)
+	if err != nil {
+		return false
+	}
+
+	for k := range csv.GetLabels() {
+		if _, ok := label[k]; ok {
```

Comment:
There are a couple of issues with this check. First: labels have the format "key=value", but you're only checking the key here, because "k" has each of the keys of the map[string]string from csv.GetLabels(). Second, this function is called "DoesOperatorHaveLabel", but the label is a map, which can actually have more than one label. Besides, the loop returns when only one of the csv's keys exists in the `label` map. I would add a "s" to the name of the function and to the label parameter, and would invert the checking order:
```
func DoesOperatorHaveLabels(..., labels map[string]string) ... {
    ...
	csvLabels := csv.GetLabels()
	for k, v := range labels {
		csvLabelValue, csvLabelExists := csvLabels[k]
		if !csvLabelExists || csvLabelValue != v {
			return false
		}
	}

	return true
}
```
This should work for whatever number of labels the parameter has. What do you think about it?

Reply:
good point, I think this makes sense